pkg/sqlcache/informer/listoption_indexer.go

```diff
@@ -905,11 +905,17 @@ func getField(a any, field string) (any, error) {
 					if !ok {
 						return nil, fmt.Errorf(failedToGetFromSliceFmt, subField, err)
 					}
-					itemStr, ok := itemVal[subField].(string)
-					if !ok {
-						return nil, fmt.Errorf(failedToGetFromSliceFmt, subField, err)
+
+					_, found := itemVal[subField]
+					if found {
+						itemStr, ok := itemVal[subField].(string)
+						if !ok {
+							return nil, fmt.Errorf(failedToGetFromSliceFmt, subField, err)
```

Comment:
`err` is null. Please drop the `%w` from `failedToGetFromSliceFmt` and remove the `err` args where it's used.

The rest is fine.

Reply:
Good catch! Fixed.